job-dsls/jobs/seed_job.groovy

```diff
@@ -51,41 +51,33 @@ job("a-seed-job") {
         shell(seedJob)
 
         jobDsl {
-            targets("job-dsls/jobs/**/communityRelease_pipeline.groovy\n" +
-                    "job-dsls/jobs/**/pr_jobs.groovy\n" +
+            targets("job-dsls/jobs/**/pr_jobs.groovy\n" +
                     "job-dsls/jobs/**/downstream_pr_jobs.groovy\n" +
                     "job-dsls/jobs/**/compile_downstream_build.groovy\n" +
-                    "job-dsls/jobs/**/downstream_production.groovy\n" +
+                    "job-dsls/jobs/**/prod_projects_downstream_production.groovy \n" +
                     "job-dsls/jobs/**/upstream.groovy\n" +
-                    "job-dsls/jobs/**/errai_pr.groovy\n" +
-                    "job-dsls/jobs/**/errai_deploy.groovy\n" +
                     "job-dsls/jobs/**/dailyBuild_pipeline.groovy\n" +
                     "job-dsls/jobs/**/dailyBuild_prod_pipeline.groovy\n" +
                     "job-dsls/jobs/**/dailyBuild_jdk11_pipeline.groovy\n" +
                     "job-dsls/jobs/**/deploy_jobs.groovy\n" +
-                    "job-dsls/jobs/**/deploy_jobs_7_x.groovy\n" +
                     "job-dsls/jobs/**/kie_jenkinsScripts_PR.groovy\n" +
                     "job-dsls/jobs/**/kie_docs_pr.groovy\n" +
                     "job-dsls/jobs/**/pr_droolsjbpm_tools.groovy\n" +
                     "job-dsls/jobs/**/prodTag_pipeline.groovy\n" +
-                    "job-dsls/jobs/**/seed_job.groovy\n" +
+                    "job-dsls/jobs/**/jenkins_shared_libs.groovy\n" +
                     "job-dsls/jobs/**/sonarcloud_daily.groovy\n" +
                     "job-dsls/jobs/**/springboot_pr_job.groovy\n" +
-                    "job-dsls/jobs/**/automatic_web_publishing.groovy\n" +
-                    "job-dsls/jobs/**/turtleTests.groovy\n" +
                     "job-dsls/jobs/**/srcclr_scan_pipeline.groovy\n" +
-                    "job-dsls/jobs/**/reduced_drools_release.groovy \n" +
-                    "job-dsls/jobs/**/prod_projects_downstream_production.groovy \n" +
                     "job-dsls/jobs/**/srcclr_scan_job.groovy\n" +
-                    "job-dsls/jobs/**/jenkins_shared_libs.groovy")
+                    "job-dsls/jobs/**/seed_job.groovy")
             useScriptText(false)
             sandbox(false)
             ignoreExisting(false)
             ignoreMissingFiles(false)
             failOnMissingPlugin(true)
             unstableOnDeprecation(true)
             removedJobAction('IGNORE')
-            removedViewAction('DELETE')
+            removedViewAction('IGNORE')
```

Comment:
why to change this attribute? just curious

Reply:
OK - we don't have views here - will fix this, mean will not change this